package.json

```diff
@@ -19,17 +19,22 @@
     "url": "git://github.com/jameslnewell/xhr-mock.git"
   },
   "dependencies": {
+    "browserify": "^11.0.1",
```

Comment:
Ok, i will try to do build into dist before publish. So move dependencies to dev again.